src/categories/models/category.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { columnSize } from '@shared/constants';
+import { BaseEntity } from '@shared/models/base.entity';
+import { index, prop } from '@typegoose/typegoose';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['index' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5693791)